ansible/roles/stack-sunbird/templates/sunbird_report-service.env

```diff
@@ -6,4 +6,5 @@ SUNBIRD_REPORTS_DB_USER={{dp_postgres_username}}
 SUNBIRD_SERVER_PORT=3030 
 SUNBIRD_BASE_REPORT_URL=report
 SUNBIRD_REPORTS_TABLE_NAME=report
+SUNBIRD_REPORT_SUMMARY_TABLE_NAME=report_summary
```

Comment:
will this value be always constant?

Reply:
both report table and report summary table are hard-coded for now.

Reply:
declare them in defaults and have it as a variable